cypress.config.js

```diff
@@ -64,7 +64,7 @@ module.exports = defineConfig({
   viewportHeight: 768,
   videoCompression: 18,
   video: true,
-  chromeWebSecurity: true,
+  chromeWebSecurity: false,
```

Comment:
-Should chromeWebSecurity be set to false for all tests globally here in config or set to false where needed in specific test? 
-Other browser like FF is not affected by this, so would this help to set it to false only on chrome?

Reply:
- It did not affect FF, only helps in Chromium based browsers.
- I noticed that partially CI hangs in Chrome, because when we navigates to other domain e.g. brb to nbc to something like that it prevent cypress navigating to the newer domain. Even though cypress sets baseUrl to other domain but it will not visit that page and cypress waiting at that point. You can see in this example run:

https://github.com/hpi-schul-cloud/e2e-system-tests/actions/runs/7634169171/job/20797659799

`Group - To show groups and classes in one table with respective functionality` feature hangs on this scenario due to the same reason and its reproducible in local as well:

`As a teacher i can not see the new class page, when the feature flag is off`

<img width="1091" alt="Screenshot 2024-01-24 at 10 13 23" src="https://github.com/hpi-schul-cloud/e2e-system-tests/assets/97288209/8f49ecdf-7e00-4637-925b-5abd582bc9b5">